### PR TITLE
chore: set default config secret name and namespace

### PR DESCRIPTION
**Pull request by nilo19**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind cleanup

**What this PR does / why we need it**:

chore: set default config secret name and namespace

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
chore: set default config secret name and namespace
```

### CODE REVIEW

File: cmd/cloud-controller-manager/app/options/dynamic.go [hunk 1]
```diff
@@ -64,8 +64,8 @@ func (o *DynamicReloadingOptions) Validate() []error {
 func defaultDynamicReloadingOptions() *DynamicReloadingOptions {
 	return &DynamicReloadingOptions{
 		EnableDynamicReloading:     false,
-		CloudConfigSecretName:      "",
-		CloudConfigSecretNamespace: "",
+		CloudConfigSecretName:      "azure-cloud-provider",
```
[the remark this answers is not in the document]
Comment (nilo19) [reply cmd/cloud-controller-manager/app/options/dynamic.go:67]: done

File: pkg/provider/azure.go [hunk 2]
```diff
@@ -373,6 +373,16 @@ func NewCloudFromConfigFile(configFilePath string, syncZones bool) (cloudprovide
 }
 
 func NewCloudFromSecret(clientBuilder cloudprovider.ControllerClientBuilder, secretName, secretNamespace, cloudConfigKey string) (cloudprovider.Interface, error) {
+	if secretName == "" {
+		secretName = consts.DefaultCloudProviderConfigSecName
+	}
+	if secretNamespace == "" {
+		secretName = consts.DefaultCloudProviderConfigSecNamespace
```
Comment (feiskyer) [inline pkg/provider/azure.go:380]: secretNamespace

File: pkg/provider/azure.go [hunk 3]
```diff
@@ -373,6 +373,16 @@ func NewCloudFromConfigFile(configFilePath string, syncZones bool) (cloudprovide
...
+	}
+	if cloudConfigKey == "" {
+		cloudConfigKey = consts.DefaultCloudProviderConfigSecKey
+	}
```
Comment (feiskyer) [inline pkg/provider/azure.go:384]: also add a unit test for these default values?

Comment (nilo19) [reply pkg/provider/azure.go:384]: adding

**Review by feiskyer — APPROVED** [review]
/lgtm

**Pull request merged**